### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by thesps**
Two small fixes, spotted preparing notebooks for FPL tutorial.
Although, the fix to the config dictionary generator is quite important.

### CODE REVIEW

File: hls4ml/utils/config.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def make_layer_config(layer):
             layer_config['Precision'] = default_precision
             layer_config['ReuseFactor'] = default_reuse_factor
             layer_config['table_size'] = 1024
-            if layer['config']['activation'] == 'softmax':
+            if layer_config['activation'] == 'softmax':
```
Comment (vloncar) [inline hls4ml/utils/config.py:147]: With the latest changes in JSON format, `Softmax` and `ReLU` are layers themselves (i.e., they can appear outside of other layers not just as `Activation` layers). They should be added to `activation_layers`

Comment (thesps) [reply hls4ml/utils/config.py]: I've added that in the latest commit.

**Review by vloncar — APPROVED** [review]
Looks good!

**Comment by thesps** [comment]
I added the new TF2.3 `Softmax` and `ReLU` layers to the supported list, and made sure the config generating utility treats both `Softmax` and `Activation(activation='softmax')` layers correctly. I did a quick test like this:
```
m = tf.keras.models.Sequential()
m.add(tf.keras.layers.Input((10,)))
m.add(tf.keras.layers.Activation(activation='softmax'))
m.add(tf.keras.layers.Activation(activation='relu'))
m.add(tf.keras.layers.Softmax())
m.add(tf.keras.layers.ReLU())
cfg = hls4ml.utils.config_from_keras_model(m, granularity='name')
hls_model = hls4ml.converters.convert_from_keras_model(m, hls_config=cfg)
hls_model.write()
```
And it all looks good.
